contrib/znc/files/znc

```diff
@@ -0,0 +1,6 @@
+type = process
+command = /usr/bin/znc -d /var/lib/znc
```

Comment:
```suggestion
command = /usr/bin/znc --datadir /var/lib/znc --foreground
```

Reply:
so change buffer to log-file?

Reply:
if it logs a bunch of stuff as it runs, probably yea

Reply:
im not sure how much it logs on the foreground and i cant test it right now

Reply:
the same as in what you already saw, but idk if it does more as things connect or something